src/common/ZulipTextButton.js

```diff
@@ -0,0 +1,109 @@
+/* @flow strict-local */
+import React, { useMemo } from 'react';
+import { View } from 'react-native';
+
+import type { LocalizableText } from '../types';
+import { BRAND_COLOR, createStyleSheet } from '../styles';
+import { Label } from '.';
+import Touchable from './Touchable';
+
+// TODO(?): Support, e.g., "small", "medium", "large" if it makes
+// sense. It probably doesn't.
+type Variant = 'standard';
+
+const styleSheetForVariant = (variant: Variant) =>
+  createStyleSheet({
+    // See https://material.io/components/buttons#specs.
+    touchable: {
+      height: 36,
+      padding: 8,
+      minWidth: 64,
+    },
+
+    // Chosen because of the value for this distance at
+    // https://material.io/components/banners#specs.
+    leftMargin: {
+      marginLeft: 8,
+    },
+    rightMargin: {
+      marginRight: 8,
+    },
+
+    // `Touchable` only accepts one child, so make sure it fills the
+    // `Touchable`'s full area and centers everything in it.
+    childOfTouchable: {
+      flex: 1,
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+
+    text: {
+      color: BRAND_COLOR,
+      textAlign: 'center',
+      textAlignVertical: 'center',
+    },
+  });
+
+type Props = $ReadOnly<{|
+  /**
+   * See note on the `Variant` type.
+   */
+  variant?: Variant,
+
+  /**
+   * Give correct in-between space for a right-aligned set of buttons.
+   */
+  leftMargin?: true,
+
+  /**
+   * Give correct in-between space for a left-aligned set of buttons.
+   */
+  rightMargin?: true,
```

Comment:
This will put a margin on the right, which is what we want if the buttons are *right*-aligned, no?

I guess it depends on whether the container already has padding such that the edge-most button needs no margin. If so, then for N buttons we only want N-1 of these spaces.

Maybe a better description would be like: "Give a right margin of the correct in-between spacing for a set of buttons."